enterprise/cloud.amazon/test/unit/src/org/netbeans/modules/cloud/amazon/AmazonInstanceTest.java

```diff
@@ -36,9 +36,10 @@ public AmazonInstanceTest() {
 
     @Test
     public void testCreateEmptyWar() throws IOException {
-        AmazonInstance ai = new AmazonInstance("dommy", "dummy", "dummy", "dummy");
+        AmazonInstance ai = new AmazonInstance("dommy", "dummy", "dummy", "dummy", "dummy");
         byte[] warData = ai.createEmptyWar();
-        Files.write(Paths.get("/home/matthias/test.zip"), warData, StandardOpenOption.CREATE, StandardOpenOption.WRITE);
+        String filePath = System.getProperty("java.io.tmpdir") + "/test.zip";
```

Comment:
Yepp, this test was not sane ... (and I introduced the hardcoded path), thanks for fixing.